timeseries/src/autogluon/timeseries/models/chronos/model.py

```diff
@@ -310,23 +312,38 @@ def _predict(
             )
             self.model_pipeline.model.eval()
             with torch.inference_mode():
-                prediction_samples = [
-                    self.model_pipeline.predict(
-                        batch,
-                        prediction_length=self.prediction_length,
-                        num_samples=self.num_samples,
-                        limit_prediction_length=False,
-                    )
-                    .detach()
-                    .cpu()
-                    .numpy()
-                    for batch in inference_data_loader
-                ]
-
-        samples = np.concatenate(prediction_samples, axis=0).swapaxes(1, 2).reshape(-1, self.num_samples)
-
-        mean = samples.mean(axis=-1, keepdims=True)
-        quantiles = np.quantile(samples, self.quantile_levels, axis=-1).T
+                if self.model_pipeline.forecast_type == ForecastType.SAMPLES:
+                    prediction_samples = [
+                        self.model_pipeline.predict(
+                            batch,
+                            prediction_length=self.prediction_length,
+                            num_samples=self.num_samples,
+                            limit_prediction_length=False,
+                        )
+                        .detach()
+                        .cpu()
+                        .numpy()
+                        for batch in inference_data_loader
+                    ]
+                    samples = np.concatenate(prediction_samples, axis=0).swapaxes(1, 2).reshape(-1, self.num_samples)
+                    mean = samples.mean(axis=-1, keepdims=True)
+                    quantiles = np.quantile(samples, self.quantile_levels, axis=-1).T
+                elif self.model_pipeline.forecast_type == ForecastType.QUANTILES:
+                    # TODO: Handle interpolation for arbitrary quantiles
+                    predictions = [
+                        self.model_pipeline.predict(
+                            batch,
+                            prediction_length=self.prediction_length,
+                            limit_prediction_length=False,
+                        )
+                        .detach()
+                        .cpu()
+                        .numpy()
+                        for batch in inference_data_loader
+                    ]
+                    model_quantiles = self.model_pipeline.quantiles
+                    quantiles = np.concatenate(predictions, axis=0).swapaxes(1, 2).reshape(-1, len(model_quantiles))
```

Comment:
We will need to implement the quantile interpolation from `model_pipeline.quantiles ` to `self.quantile_levels` for the unit tests to pass

Reply:
I'm fine with just clipping to the max quantile available for the model for extrapolation. Have we tried training the base models with the [0.05, 0.10, ..., 0.95] quantile levels?

Reply:
No we didn't. I think clipping is quite a silent side effect, should we warn/err out instead?

Reply:
Warning in the logs is fine, I don't think that we should error since this will completely remove the model in case the user cares about high/low quantiles.